src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,83 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final byte RESIZE = 2;
+    private int size;
+    private Node[] table;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resize();
+        }
+        int index = hashCode(key);
+        Node node = table[index];
```

Comment:
```suggestion
        Node<K, V> node = table[index];
``` 
дженеріки використовуємо там де вказуємо тип змінної, а там де відставляємо значення, варто використовувати просто кутові дужки '<>'
тут детальніше про це можна почитати https://www.baeldung.com/java-diamond-operator